example/src/LinearGradientExample.js

```diff
@@ -1,6 +1,7 @@
 import * as React from "react";
 import { Text } from "react-native";
-import { ScreenContainer, withTheme, LinearGradient } from "@draftbit/ui";
+import { ScreenContainer, withTheme } from "@draftbit/ui";
```

Comment:
needs to exist from UI, just exported from native instead of core

Reply:
@peterpme Gotcha.